crates/miden-objects/src/account/partial.rs

```diff
@@ -67,6 +71,51 @@ impl PartialAccount {
     pub fn vault(&self) -> &PartialVault {
         &self.partial_vault
     }
+
+    /// Returns true if the account is new (i.e. its nonce is zero and it hasn't been registered on
+    /// chain yet).
+    pub fn is_new(&self) -> bool {
+        self.nonce == ZERO
+    }
+
+    /// Returns the commitment of this account.
+    ///
+    /// The commitment of an account is computed as:
+    ///
+    /// ```text
+    /// hash(id, nonce, vault_root, storage_commitment, code_commitment).
+    /// ```
+    pub fn commitment(&self) -> Word {
+        hash_account(
```

Comment:
nit: maybe the function name here should be something like `compute_account_commitment()`?

Reply:
I think this is basically this discussion: https://github.com/0xMiden/miden-base/pull/1840#discussion_r2317883738, i.e. it should be `to_commitment` long-term, but for now `commitment` matches `Account::commitment`.